core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -221,7 +226,7 @@ public void testFromXContent() throws IOException {
      */
     @Test
     public void testToQuery() throws IOException {
-        QueryParseContext context = createContext();
+        QueryCreationContext context = createContext();
```

Comment:
I would rename this createContext method too once we are settled on the naming for QueryCreationContext